src/Label.tsx

```diff
@@ -54,7 +54,7 @@ const Label = styled.span<
 >`
   display: inline-block;
   font-weight: ${get('fontWeights.semibold')};
-  color: ${get('colors.white')};
+  color: ${get('colors.text.inverse')};
   border-radius: ${get('radii.3')};
 
   &:hover {
```

Comment:
Commenting here because I'm not sure how to comment on collapsed lines! L66 has a shadow value with a color but I can't find any shadow that would correspond to it - one thing I'm starting to notice is there are places that could benefit from upstream changes to primitives... is that something we want to consider?

Reply:
@VanAnderson for now, I think these variable choices are fine.